api/router.go

```diff
@@ -93,6 +93,11 @@ func NewRouter(s *Server) *mux.Router {
 		"snapshotRevert": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotRevert),
 		"snapshotBackup": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotBackup),
 
+		"snapshotCRCreate": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotCRCreate),
+		"snapshotCRList":   s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotCRList),
+		"snapshotCRGet":    s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotCRGet),
+		"snapshotCRDelete": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotCRDelete),
+
```

Comment:
Will we remove original snapshotList/snapshotGet/snapshotDelete/snapshotCreate methods?

Reply:
No, we don't remove them because they are needed for getting a valid snapshot chain at a point in time

Reply:
Why cannot we directly change the existing snapshot CRUD APIs for the snapshot CR introduction?